Client/Frontend/UIConstants.swift

```diff
@@ -35,6 +35,6 @@ public struct UIConstants {
     static let PasscodeEntryFont = UIFont.systemFont(ofSize: PasscodeEntryFontSize, weight: UIFont.Weight.bold)
 
     /// JPEG compression quality for persisted screenshots. Must be between 0-1.
-    static let ScreenshotQuality: Float = 0.3
+    static let ScreenshotQuality: Float = 0.03
```

Comment:
I don't know that this is the best way to improve memory used by the screenshots as JPEG's can look very grainy at that low compression quality, plus it will only save storage space and not much memory (if any) as it's converted back into a full size bitmap when loaded into memory.
Instead can you reduce the resolution of the image that is stored to be closer to the size of the image that is used in the tab tray cell?

Reply:
Curious 🤔 How are you measuring the data size for images?

Reply:
by adding a breakpoint at if let data = image.jpegData(compressionQuality: self.quality) and checking the size of the data object

Reply:
That will tell you the size of the data file that is written to disk, not the amount of ram it takes up in memory when it's loaded for rendering. A jpeg is heavily compressed to optimize for storage, using the jpeg algorithm it doesn't need to store a value for every single pixel so it can massively reduce it's size when saved to disk. When loaded in memory for rendering it's essentially stored as a 2D array with one element for ever pixel, the size of that element depends on the number of channels, typically RBGA. So the memory footprint of the image is a function of it's resolution and the number of channels needed to express each pixel.

Reply:
I did a test yesterday and I can't see any difference but that's kind of expected as I'm colorblind but I'll leave a screenshot in case the difference is obvious to normal vision people so I'll leave it for reference in case we'd need to get back to this discussion
![image](https://user-images.githubusercontent.com/126153378/226855618-2bff1506-6a9f-4fd8-a81c-f9fe5727a5b7.png)